src/app/modules/main/module.nim

```diff
@@ -519,6 +519,9 @@ method communityJoined*[T](
   messageService: message_service.Service,
   gifService: gif_service.Service,
 ) =
+  var firstCommunityJoined = false
+  if (self.communitySectionsModule.len == 0):
```

Comment:
@jrainville since you need this check only in a single place (line 540) you don't need to introduce `firstCommunityJoined` as a new variable, just use this check there.

Reply:
Ah true, I could have done `if (self.communitySectionsModule.len == 1):` instead. I did it with the variable because I knew `communitySectionsModule` was modified below, but obviously `0 + 1 == 1` 😆